### ISSUE TITLE
NumPy 1.24 breaks pupil/gaze tracking

### ISSUE THREAD

**Issue by X-m7** [emilianavt/OpenSeeFace#53]
NumPy 1.24 appears to break pupil/gaze tracking, when running `python facetracker.py --model 4 -c models/benchmark.bin --repeat-video 1 -v 1` there is no point in or near the pupil, but after downgrading to NumPy 1.23 the dot tracking the pupil appears again.

Below is the output with NumPy 1.23 and 1.24 for comparison:
![numpy123](https://user-images.githubusercontent.com/18729369/227669497-ab41444a-c911-4ee3-8e62-6700fcc74135.png)
![numpy124](https://user-images.githubusercontent.com/18729369/227669496-d83c66b3-4896-4b2d-9fbe-875f5df2a854.png)

I am using Python 3.10.10 on Arch Linux, and the specific NumPy versions I tried are 1.23.5 and 1.24.2.

Output of `pip freeze` in the virtual environment excluding `numpy`:
```
coloredlogs==15.0.1
flatbuffers==23.3.3
humanfriendly==10.0
mpmath==1.3.0
onnxruntime==1.14.1
opencv-python==4.7.0.72
packaging==23.0
Pillow==9.4.0
protobuf==4.22.1
sympy==1.11.1
```

P.S. `onnxruntime` works for Python 3.10 now so no need to use the nightly version.

**Comment by emilianavt** [comment]
Scary, I wonder how that is happening.

**Comment by emilianavt2** [comment]
This should be fixed by #57.
